modules/spo-replicating-controllers.adoc

```diff
@@ -1,24 +1,12 @@
 // Module included in the following assemblies:
 //
-// * security/security_profiles_operator/spo-seccomp.adoc
 // * security/security_profiles_operator/spo-selinux.adoc
 
-ifeval::["{context}" == "spo-seccomp"]
-:seccomp:
-:type: seccomp
-:kind: SeccompProfile
-endif::[]
-ifeval::["{context}" == "spo-selinux"]
-:selinux:
-:type: SELinux
-:kind: SelinuxProfile
-endif::[]
-
 :_mod-docs-content-type: PROCEDURE
 [id="spo-replicating-controllers_{context}"]
 = Replicating controllers and SecurityContextConstraints
 
-When deploying {type} policies for replicating controllers, such as deployments or daemon sets, note that the `Pod` objects spawned by the controllers are not running with the identity of the user who creates the workload. Unless a `ServiceAccount` is selected, the pods might revert to using a restricted `SecurityContextConstraints` (SCC) which does not allow use of custom security policies.
+When deploying SELinux policies for replicating controllers, such as deployments or daemon sets, note that the `Pod` objects spawned by the controllers are not running with the identity of the user who creates the workload. Unless a `ServiceAccount` is selected, the pods might revert to using a restricted `SecurityContextConstraints` (SCC) which does not allow use of custom security policies.
```

Comment:
```suggestion
When you deploy SELinux policies for replicating controllers, such as deployments or daemon sets, note that the `Pod` objects spawned by the controllers are not running with the identity of the user who creates the workload. Unless a `ServiceAccount` is selected, the pods might revert to using a restricted `SecurityContextConstraints` (SCC) which does not allow use of custom security policies.
```
- I replaced "When deploying" with "When you deploy" per IBM Style's guidance about avoiding dangling modifiers: https://www.ibm.com/docs/en/ibm-style?topic=grammar-verbs#general-guidelines
- The terms `ServiceAccount` and `SecurityContextConstraints` are formatted as though they are objects, resources, or specs. For clarity, consider adding the appropriate noun after each term (i.e., "Unless a `ServiceAccount` object is selected...").